### PR TITLE
Team Get Info

### PR DESCRIPTION
**Pull request by nboricic**
Translations and examples regarding /team/info endpoint

### CODE REVIEW

**Comment by TaylorKrusen** [comment]
Failling at OAS validation. You can run the linter locally with `openapi lint openapi.yaml`

File: examples/TeamInfo.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const api = new HelloSignSDK.TeamApi();
+
+// Configure HTTP basic authorization: api_key
+api.username = "YOUR_API_KEY";
+
+// or, configure Bearer (JWT) authorization: oauth2
+// $config->setAccessToken("YOUR_ACCESS_TOKEN");
+
+const result = api.teamInfo("team_id");
```
Comment (TaylorKrusen) [inline examples/TeamInfo.ts:12]: Was this sample generated by the typescript SDK? 
I'd expect this team id parameter to be broken out into it's own type. Something like this: 
```json
const data: HelloSignSDK.TeamGetInfoRequest = {
  team_id: "21651sdf5161",
};
```

**Review by TaylorKrusen — COMMENTED** [review]
Still needs: 
- [json] TeamGetInfoRequestExample and TeamGetRootInfoRequestExample
- a "requestBody" section under the path in the OAS
- a request schema for the endpoint

**Review by TaylorKrusen — APPROVED** [review]
LGTM

**Review by armandogomez0 — APPROVED** [review]

**Pull request merged**